src/internal/zod.ts

```diff
@@ -30,6 +30,7 @@ import type {
 	ZodOptional as TrueZodOptional,
 	ZodPipeline as TrueZodPipeline,
 	ZodPromise as TrueZodPromise,
+	ZodReadonly as TrueZodReadonly,
```

Comment:
That's a great question. I haven't tried it. In terms of the actual code, it shouldn't cause any problems. The only thing that ships in our compiled source are the mocks, which means it doesn't matter if you have ZodReadonly or not, the mock will always be there. The thing I'm not sure about is whether it would cause a TS error.

Reply:
Everything seem OK on my end 👍